requirements.txt

```diff
@@ -0,0 +1,6 @@
+-e .
```

Comment:
what is this file used for ? test dependencies is defined in `tox.ini`

Reply:
For simplicity. But, let's skip this then, I've removed the file as it would require more refactor to share single requirements in multiple cases.